src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -1011,6 +1011,23 @@ def rename_symbol(self, symbol, name):
         # Re-insert modified symbol
         self.add(symbol)
 
+    def get_first_wildcard_import(self):
```

Comment:
I was thinking about this after I put it in for review. In the longer term we may want the full list of containers with wildcard imports if we're trying to chase down the definition of a particular symbol. However, probably I should simplify the implementation so that it just does what is currently required.